wpk/README.md

```diff
@@ -3,68 +3,10 @@ WPK package
-    ```
-    Set the location as follows:
-
-    - /C=US is the country.
-    - /ST=CA is the state.
-    - /O=Wazuh is the organization’s name.
-
-    ```shellsession
-    # openssl x509 -req -days 365 -in wpkcert.csr -CA wpk_root.pem -CAkey wpk_root.key -out wpkcert.pem -CAcreateserial
-    ```
-3. Execute the `generate_wpk_package.sh` script to build the package. There are multiple parameters to select, Here you can see all the different parameters:
-    ```shellsession
-    $ ./generate_wpk_package.sh -h
-
-    Usage: ./generate_wpk_package.sh [OPTIONS]
-
-        -t,   --target-system <target>              [Required] Select target wpk to build [linux/windows]
-        -b,   --branch <branch>                     [Required] Select Git branch or tag e.g.
-        -d,   --destination <path>                  [Required] Set the destination path of package.
-        -k,   --key-dir <path>                      [Required] Set the WPK key path to sign package.
-        -a,   --architecture <arch>                 [Optional] Target architecture of the package [x86_64].
-        -j,   --jobs <number>                       [Optional] Number of parallel jobs when compiling.
-        -pd,  --package-directory <directory>       [Required for windows] Package name to pack on wpk.
-        -o,   --output <name>                       [Required] Name to the output package.
-        -c,   --checksum                            [Optional] Generate checksum.
-        -h,   --help                                Show this help.
-    ```
-    * To build the WPK package for linux called linux-3_9_0.wpk from tag v3.9.0 and store it in /home/wpk, while having the keys stored in /tmp/keys
-
-        ```shellsession
-        # ./generate_wpk_package.sh -t linux -b v3.9.0 -d /home/wpk -k /tmp/keys -o linux-3_9_0.wpk
-        ```
-    * To build the WPK package for linux called linux-3_8.wpk from branch 3.8 and store it in /home/wpk, while having the keys stored in /tmp/keys
-
-        ```shellsession
-        # ./generate_wpk_package.sh -t linux -b 3.8 -d /home/wpk -k /tmp/keys -o linux-3_8.wpk
-        ```
-    For windows packages it is necessary to have a msi package, you can find the latest packages here: [wazuh packages](https://documentation.wazuh.com/current/installation-guide/packages-list/index.html#packages)
-
-    * To build the WPK package for windows called windows-3_9_0.wpk from tag v3.9.0 and store it in /home/wpk, while having the keys stored in /tmp/keys and the msi stored in /tmp/msi/wazuh-agent-3.9.0-1.msi
-        ```shellsession
-        # /generate_wpk_package.sh -t windows -b v3.9.0 -d home/wpk -k /tmp/keys -o windows-3_9_0.wpk -pd /tmp/msi/wazuh-agent-3.9.0-1.msi
-        ```
-    3. Once the execution finishes, you will find your `.zip` package in the specified folder.
+All info is in [Generate-wpk-package.](https://documentation.wazuh.com/3.11/packaging/generate-wpk-package.html)
```

Comment:
Please, replace "All info is in" by "Please, visit the following link for the full WPK packages building documentation:"

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.